src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/GoalEntity.kt

```diff
@@ -48,6 +50,13 @@ class GoalEntity(
   @Column(name = "creation_date")
   val creationDate: String = DateTimeFormatter.ISO_INSTANT.format(Instant.now()),
 
+  @Column(name = "goal_status")
+  @Enumerated(EnumType.STRING)
+  var goalStatus: GoalStatus? = null,
```

Comment:
Could we just call this `status`? I don't think we need to prepend `goal`, as it's part of the Goal entity

Reply:
amended